bindata/v4.1.0/config/defaultconfig.yaml

```diff
@@ -61,13 +61,16 @@ auditConfig:
       resources:
       - group: "oauth.openshift.io"
         resources: ["oauthaccesstokens", "oauthauthorizetokens"]
-    # Don't log authenticated requests to certain non-resource URL paths.
+    # Don't log requests to /healthz from kubelets
+    - level: None
+      userGroups: ["system:nodes"]
+      nonResourceURLs:
+        - "/healthz"
+    # Don't log requests to certain non-resource URL paths.
     - level: None
-      userGroups: ["system:authenticated", "system:unauthenticated"]
       nonResourceURLs:
       - "/api*" # Wildcard matching.
```

Comment:
how it is possible that it doesn't match `/apis/group/version`?

Reply:
because of nonResourceURLs. The actual APIs are resource URLs.